nomad/state/paginator/paginator.go

```diff
@@ -54,12 +57,16 @@ func NewPaginator(iter Iterator, tokenizer Tokenizer, filters []Filter,
 		filters = append(filters, evaluator)
 	}
 
+	// attempt to convert token to uint for iterators ordered numerically
+	seekingUint, _ := strconv.ParseUint(opts.NextToken, 10, 64)
```

Comment:
Dropping errors is "weird" enough we should report why. Something like:

```suggestion
	// attempt to convert token to uint for iterators ordered numerically.
	// it's safe to ignore the error here because the `next` method ignores 
	// this field for string tokens and 0 is valid for an unset numeric NextToken
	seekingUint, _ := strconv.ParseUint(opts.NextToken, 10, 64)
```